modules/standard/IO.chpl

```diff
@@ -6175,71 +6175,12 @@ proc file.readerHelper(param kind=_iokind.dynamic, param locking=true,
   return ret;
 }
 
-@deprecated("lines with a 'local_style' argument is deprecated")
-proc file.lines(param locking:bool = true, start:int(64) = 0,
-                end:int(64) = max(int(64)), hints=ioHintSet.empty,
-                in local_style:iostyle) throws {
-  return this.linesHelper(locking, start..end, hints,
-                          local_style: iostyleInternal);
-}
-
-/* Used to control the behavior of the region argument for :proc:`file.lines`.
+/* Used to control the behavior of the region argument for :proc:`file.lines`. // which has now been removed
```

Comment:
Should we just remove this as well?

Reply:
This config param was only deprecated for 1.32, so I want to leave it for at least another release in case people have the flag in their makefiles, etc.